src/web/mjs/connectors/NineEkor.mjs

```diff
@@ -13,15 +13,15 @@ export default class NineEkor extends Connector {
 
     async _getMangaFromURI(uri) {
         let request = new Request(uri, this.requestOptions);
-        let data = await this.fetchDOM(request, 'div.page-head h1.page-title');
-        let id = uri.pathname.split('page/')[0];
+        let data = await this.fetchDOM(request, 'main.site-main header.page-header h1.entry-title');
+        let id = uri;
```

Comment:
Do not use absolute URL, except the website uses cross-domain links
`uri.pathname`